### DISCUSSION TITLE
No tagged releases since October 2022 ?

### DISCUSSION THREAD

**Discussion by icj217** [copier-org/copier#1058] [General]
There hasn't been any new tagged releases since `v7.0.1` back in October 2022, despite the fact that there's been a bunch of bug fixes/enhancements merged into `master` in the interim. Is there any reason for this? Can we get a new tagged release? Otherwise there's no real way to get access to these improvements when installing copier from PyPi

cc: @yajo

**Comment by yajo** [comment]
You can see this milestone:
https://github.com/copier-org/copier/milestone/16

I plan to do a new release once the list is clear. It's close!

**Comment by sisp** [comment]
@icj217 Copier v7.1.0 was released yesterday.

**Comment by icj217** [comment]
Thanks all!
